contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -2120,11 +2120,15 @@ END;
 $$ LANGUAGE 'pltsql';
 GRANT EXECUTE ON PROCEDURE sys.sp_linkedservers TO PUBLIC;
 
+SET allow_system_table_mods = on;
```

Comment:
so we are only ignoring DDLs but not DMLs? will we propagate DMLs or they will be ignored via other mechanism?

Reply:
Not sure if I understood your question but the use of `allow_system_table_mods` is to allow DDLs on bbf catalog tables. The DML propagation(skipped) will be skipped by marking them catalog tables, see `IsPLtsqlExtendedCatalog` below.